bpf-filter/test-bpf.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
I'd suggest always adding
```sh
set -o errexit
set -o nounset
```

to your shell scripts. The former will make sure things error out early if something is wrong, so you don't accidentally continue with wrong assumptions. And the latter protects you against using unset variables which expand to bad things (the worst example of course being `rm -rf /$MY_UNSET_VAR`...).

In this case I initially forgot I had not enabled sch_htb in my kernel config, and the script ran all the way to iperf without complaining...

Reply:
That's perfect. I will add that to the scripts. :)